triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -853,6 +855,13 @@ private[lf] class Runner private (
               "Trigger rule initial state",
               "state" -> triggerUserState(state, trigger.defn.level),
             )
+            if (trigger.defn.level == Trigger.Level.High) {
```

Comment:
For low level triggers, we log the entire state so there's no need to log internal state